### PR TITLE
Fix empty NuGets

### PR DESCRIPTION
**Pull request by piotrpMSFT**
Fixes issue #797 
Builds all to-be-packaged assemblies, fixes packaging script.

### CODE REVIEW

**Comment by davidfowl** [comment]
:shipit:

**Comment by livarcocc** [comment]
:shipit:

**Comment by anurse** [comment]
Is this ready to be merged? @piotrpMSFT ?

**Comment by piotrpMSFT** [comment]
@dotnet-bot test this please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test this please.

**Comment by livarcocc** [comment]
@dotnet-bot Test Ubuntu Debug Build Please
@dotnet-bot Test Ubuntu Release Build Please

File: src/Microsoft.Extensions.DependencyModel/project.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   },
   "frameworks": {
     "net451": { },
-    "dotnet5.4": {
+    "dnxcore50": {
```
Comment (anurse) [inline src/Microsoft.Extensions.DependencyModel/project.json]: Why was this change necessary? We need to use this library in Mvc, which targets `dotnet5.4`, not `dnxcore50`. Also, we should start moving away from `dnxcore50` anyway...

Comment (anurse) [inline src/Microsoft.Extensions.DependencyModel/project.json]: + @pakrym

Comment (pranavkm) [inline src/Microsoft.Extensions.DependencyModel/project.json]: This is blocking Mvc from consuming this package. I'm going to try hand editing it for now, but dotnet5.x should be the way to go.

Comment (anurse) [inline src/Microsoft.Extensions.DependencyModel/project.json]: #829 should have resolved this now.
